packages/itwinui-react/src/core/utils/components/PolymorphicAction.tsx

```diff
@@ -0,0 +1,44 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import {
+  PolymorphicComponentProps,
+  PolymorphicForwardRefComponent,
+} from '../index';
+
+type PolymorphicActionOwnProps = {
+  /**
+   * Content in the action element.
+   */
+  children?: React.ReactNode;
```

Comment:
`children` doesn't need to be a special prop.

Reply:
It does not work without children so I kept it in props.

Reply:
what doesn't work without children? if you check `Flex.tsx`, none of them have explicitly defined children.

i think you can just use `{}`

Reply:
I can use `{}` with eslint disabled

Reply:
`{}` can be assigned to a different type for better readability
![](https://user-images.githubusercontent.com/9084735/223806838-5896f60b-0342-4fa1-b096-9aecb2a17194.png)

this also makes it easier to add more props later if needed

Reply:
Using own props objects.

Reply:
you can remove these eslint comments, they are only needed in one place.

![](https://user-images.githubusercontent.com/9084735/225352349-f8787a87-97d3-463b-bb57-c8b4a5635777.png)

Reply:
Oops